### DISCUSSION TITLE
Packaging Efforts

### DISCUSSION THREAD

**Discussion by cafkafk** [eza-community/eza#108] [Development]
Since most packaging efforts happens outside of the eza repository, I thought it would be nice to have a common thread to track efforts and coordinate, as well as ask any questions related to packaging.

Current status:
- [x] NixOS (nixpkgs) https://github.com/NixOS/nixpkgs/pull/246050
- [x] AUR (arch user repository) https://aur.archlinux.org/packages/eza-git 
- [x] Crates.io https://crates.io/crates/eza
- [x] .deb https://github.com/eza-community/eza/issues/143

[![Packaging status](https://repology.org/badge/vertical-allrepos/eza.svg)](https://repology.org/project/eza/versions)

In most cases, it should be trivial to create package definitions from those used for exa.

**Comment by Fryboyter** [comment]
I would have two comments on the package in the AUR.

For the creation the dependency pandoc-cli is currently specified. This package in turn requires some Haskell packages. Under Arch there are very often updates for Hasekell packages. And always for all of them. 

I, and probably many other users, therefore prefer pandoc-bin from the AUR, because with it you don't need the Hasekell packages. So in my opinion it would make more sense to specify pandoc instead of pandoc-cli as a dependency. Then you have the possibility to choose between pandoc-cli and pandoc-bin during the installation. This is how it is done with the package exa (https://gitlab.archlinux.org/archlinux/packaging/packages/exa/-/blob/main/PKGBUILD#L14).

Eza is currently offered in the AUR as a Git version and is therefore always up-to-date. However, eza-git cannot be installed in the AUR because the file maneza.1.md is not found. This is probably because the man pages have been renamed (https://github.com/eza-community/eza/commit/c92602d55a892743b8ffc58d0391d06555482e33).

In the AUR I have already created a corresponding comment (https://aur.archlinux.org/packages/eza-git#comment-930104).

From my point of view it would also make sense if a stable version (e.g. 0.10.7) is offered in addition to eza-git. Because not everyone wants to use git versions.

**Reply by gierens** [reply]
Maybe @zefr0x who maintains the AUR package can give more insights on this, but if I'm not mistaken it does already take the stable version, see the `git reset --hard "$(git describe --tags --abbrev=0 --match 'v*')"` in the `PKGBUILD` of `eza-git`.

**Reply by zefr0x** [reply]
Yes, it does already take the latest stable version.

About the other issues I provided a response comment in the AUR.
